sshd-sftp/src/main/java/org/apache/sshd/sftp/server/AbstractSftpSubsystemHelper.java

```diff
@@ -338,6 +338,9 @@ protected void process(Buffer buffer) throws IOException {
             if (type == SftpConstants.SSH_FXP_INIT) {
                 throw e;
             }
+            if (log.isDebugEnabled()) {
+                log.debug("process({}) Exception", session, e);
+            }
```

Comment:
Why don't you do this _inside_ `sendStatus(Buffer buffer, int id, Throwable e, int cmd, Object... args)`? There would be only a single place to change. The exception stack trace that would be logged would identify where it occurred.

Reply:
The method name will get lost.    As it should be on the stack trace it should be okay.    I made the change,  please review.